Source/Manager/User/User+load.swift

```diff
@@ -14,4 +14,20 @@ public extension User {
         try? user.loadStoredTokens()
         return user
     }
+    /**
+     Enables App transfer step 1
+     */
+    static func storeOnDevice(withConfiguration clientConfiguration: ClientConfiguration, storageKey: String) {
+        let user = User(clientConfiguration: clientConfiguration)
+        try? user.storeOnDevice(key: storageKey)
```

Comment:
Hiding the error here seems like a poor choice. The apps won't know it failed, and won't retry to store the key again.

Reply:
Sure. I'll propagate up the error to the apps 👌